lib/mjml/parser.rb

```diff
@@ -64,11 +74,28 @@ def validation_level
       "--level=#{MJML.config.validation_level}" if MJML::Feature.available?(:validation_level)
     end
 
+    def output_from_file
+      @temp_file = Tempfile.new("mjml-template")
+      _out, err, _sts = Open3.capture3(cmd, stdin_data: @template)
+      @temp_file.rewind
+      @temp_file.unlink
+      return @temp_file.read, err
+    end
+
+    def output_from_memory
+      out, err, _sts = Open3.capture3(cmd, stdin_data: @template)
+      return out, err
+    end
+
+    def should_get_outpout_from_file?
+      @template.size > 20_000
```

Comment:
Actually, the size of my template was 27000. But the size that really import here is the size of the output. So, the 20000 number is not random, it's a estimation according to my different tests. I'm trying to avoid writing files for templates that are small.